swagger.json

```diff
@@ -2527,7 +2743,7 @@
       },
       "get": {
         "summary": "Cancels any back-end computations of a job",
-        "description": "For batch jobs this request cancels all related computations at the back-end. Lazy jobs will not respond to any further on demand requests, e.g. download calls or service requests. This job won't generate additional costs for processing. Results of batch jobs might be discarded and deleted by the back-end.",
+        "description": "For batch jobs this request cancels all related computations at the back-end. Lazy jobs will not respond to any further on demand requests, e.g. download calls or service requests. This job won't generate additional costs for processing unless started again using `GET /jobs/{job_id}/queue` (for batch jobs) or updated using `PATCH /jobs`. Results of batch jobs might be discarded and deleted by the back-end.",
```

Comment:
We didn't have a solution to "restart" a lazy job. Does this clarification sound useful?
So restarting a lazy job could be done with a (maybe empty) patch.